<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="localRegression" duration-ms="5974" started-at="2020-02-16T14:00:19 EET" finished-at="2020-02-16T14:00:25 EET">
    <groups>
      <group name="groupCreation">
        <method signature="AdminCreateGroupTest.testToCreateNewGroup(resources.models.Group, java.lang.String, java.lang.String)[pri:1, instance:com.TheGoodGuys.DMS1.AdminCreateGroupTest@57855c9a]" name="testToCreateNewGroup" class="com.TheGoodGuys.DMS1.AdminCreateGroupTest"/>
      </group> <!-- groupCreation -->
      <group name="invalidUserLogin">
        <method signature="InvalidLoginTest.testInvalidUserLogin(java.lang.String, java.lang.String)[pri:1, instance:com.TheGoodGuys.DMS1.InvalidLoginTest@3b084709]" name="testInvalidUserLogin" class="com.TheGoodGuys.DMS1.InvalidLoginTest"/>
      </group> <!-- invalidUserLogin -->
    </groups>
    <test name="GroupCreationTest" duration-ms="5974" started-at="2020-02-16T14:00:19 EET" finished-at="2020-02-16T14:00:25 EET">
      <class name="com.TheGoodGuys.DMS1.AdminCreateGroupTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.TheGoodGuys.DMS1.AdminCreateGroupTest@57855c9a]" name="setUp" is-config="true" duration-ms="1764" started-at="2020-02-16T14:00:19 EET" finished-at="2020-02-16T14:00:21 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="preconditions(java.lang.String)[pri:0, instance:com.TheGoodGuys.DMS1.AdminCreateGroupTest@57855c9a]" name="preconditions" is-config="true" duration-ms="1155" started-at="2020-02-16T14:00:21 EET" finished-at="2020-02-16T14:00:22 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://localhost:3000/#/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preconditions -->
        <test-method status="FAIL" signature="testToCreateNewGroup(resources.models.Group, java.lang.String, java.lang.String)[pri:1, instance:com.TheGoodGuys.DMS1.AdminCreateGroupTest@57855c9a]" name="testToCreateNewGroup" duration-ms="219" started-at="2020-02-16T14:00:22 EET" data-provider="validGroups" finished-at="2020-02-16T14:00:22 EET">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void com.TheGoodGuys.DMS1.AdminCreateGroupTest.testToCreateNewGroup(resources.models.Group,java.lang.String,java.lang.String) throws java.lang.Exception] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testToCreateNewGroup([Parameter{index=0, type=resources.models.Group, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(resources.models.Group) resources.models.Group@4dc8caa7]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.TheGoodGuys.DMS1.AdminCreateGroupTest.testToCreateNewGroup(resources.models.Group,java.lang.String,java.lang.String) throws java.lang.Exception] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testToCreateNewGroup([Parameter{index=0, type=resources.models.Group, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=2, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(resources.models.Group) resources.models.Group@4dc8caa7]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:925)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testToCreateNewGroup -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.TheGoodGuys.DMS1.AdminCreateGroupTest@57855c9a]" name="closeBrowser" is-config="true" duration-ms="80" started-at="2020-02-16T14:00:22 EET" finished-at="2020-02-16T14:00:23 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.TheGoodGuys.DMS1.AdminCreateGroupTest -->
      <class name="com.TheGoodGuys.DMS1.InvalidLoginTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.TheGoodGuys.DMS1.InvalidLoginTest@3b084709]" name="setUp" is-config="true" duration-ms="1699" started-at="2020-02-16T14:00:23 EET" finished-at="2020-02-16T14:00:24 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="preconditions(java.lang.String)[pri:0, instance:com.TheGoodGuys.DMS1.InvalidLoginTest@3b084709]" name="preconditions" is-config="true" duration-ms="750" started-at="2020-02-16T14:00:24 EET" finished-at="2020-02-16T14:00:25 EET">
          <params>
            <param index="0">
              <value>
                <![CDATA[http://localhost:3000/#/]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preconditions -->
        <test-method status="FAIL" signature="testInvalidUserLogin(java.lang.String, java.lang.String)[pri:1, instance:com.TheGoodGuys.DMS1.InvalidLoginTest@3b084709]" name="testInvalidUserLogin" duration-ms="16" started-at="2020-02-16T14:00:25 EET" data-provider="invalidUser" finished-at="2020-02-16T14:00:25 EET">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void com.TheGoodGuys.DMS1.InvalidLoginTest.testInvalidUserLogin(java.lang.String,java.lang.String) throws java.lang.Exception] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testInvalidUserLogin([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(resources.models.User) resources.models.User@63fdab07]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.TheGoodGuys.DMS1.InvalidLoginTest.testInvalidUserLogin(java.lang.String,java.lang.String) throws java.lang.Exception] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: testInvalidUserLogin([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}, Parameter{index=1, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(resources.models.User) resources.models.User@63fdab07]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:925)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidUserLogin -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.TheGoodGuys.DMS1.InvalidLoginTest@3b084709]" name="closeBrowser" is-config="true" duration-ms="135" started-at="2020-02-16T14:00:25 EET" finished-at="2020-02-16T14:00:25 EET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.TheGoodGuys.DMS1.InvalidLoginTest -->
    </test> <!-- GroupCreationTest -->
  </suite> <!-- localRegression -->
</testng-results>
